template/package.json

```diff
@@ -19,8 +19,8 @@
     "node": ">=[nodeVersion]"
   },
   "dependencies": {
-    "chalk": "^4.1.2",
-    "lit": "^2.7.2"
+    "chalk": "",
```

Comment:
As above, shouldn't this just be a dev dep?

Reply:
No, this runs on the client using the generator, so it's needed there to work.